#goal definition
uint16 beverage_type
---
#result definition
uint16 result
---
#feedback
uint16 mechanismStep